content/docs/app-developer-guide/specific-buildpacks.md

```diff
@@ -4,19 +4,29 @@ weight=4
 summary="Learn how to specify exactly what buildpacks are used during the build process."
 +++
 
-{{< param "summary" >}}
+You may specify exactly what buildpacks are used during the build process by referencing them with a URI in any of the following formats.
 
-### `--buildpacks`
+| Type | Format |
```

Comment:
I was going to argue that this was Pack specific and should be noted as such but given that we want support for buildpack URIs in the project descriptor which are portable it makes me wonder if buildpack URIs should be spec'd since they we will be more endemic than we originally thought. 

I can see them as part of the [distribution](https://github.com/buildpacks/spec/blob/main/distribution.md) spec but maybe an extension spec is more appropriate.

Thoughts?

CC: @elbandito

Reply:
Yea, I agree. I'll draft an extension spec, and we can decide if it should move into distribution (which I agree makes the most sense)